cmd/generate/dockerfile-templates/MustGatherDockerfile.template

```diff
@@ -0,0 +1,25 @@
+# DO NOT EDIT! Generated Dockerfile for {{.main}}.
+ARG MUSTGATHER={{.must_gather_base}}
+ARG RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
+FROM $MUSTGATHER AS builder
+
+FROM $RUNTIME
+
+COPY --from=builder /usr/bin/oc /usr/bin/oc
+COPY --from=builder /usr/bin/gather /usr/bin/gather_original
+
+# Copy all collection scripts to /usr/bin
+COPY bin/* /usr/bin/
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-{{.project_dashcase}}rhel8-container" \
```

Comment:
@creydr rhel version, shoudn't it be injected ?

Reply:
maybe as a follow up as I see it on other templates

Reply:
Good catch! I agree, that we can do this as a follow up, as this needs to be addressed in the other templates too

Reply:
@creydr can we merge then?

Reply:
I think we need the rpmlock file: https://github.com/openshift-knative/hack/pull/370#discussion_r1827283436

Reply:
Right. and https://github.com/openshift-knative/hack/pull/370/files#r1827520465 (which I commented earlier on the PR in s-o, but forgot here, so added now)

Reply:
Done, i have added back rsync and tar.